### ISSUE TITLE
remove silly words: POG, POGCHAMP, THOOMIN, etc

### ISSUE THREAD

**Issue by iminlikewithyou** [omg-community/vivi#10]
these are quite dumb

**Comment by KisaragiDayo** [comment]
Would you keep Roblox, Robloxian, Robux?

**Comment by iminlikewithyou** [comment]
yes, as specialized words #11 

https://github.com/omg-community/vivi/pull/9#pullrequestreview-1827280102

**Comment by Shrenne** [comment]
Pog and pogchamp are used as interjections expressing something satisfactory and commonly used as a word detached from the emoji they were based on. I think they have a good reason to be in.

**Comment by KisaragiDayo** [comment]
And also both of these words are widely used enough by people to the point that it'd be viable to added/stay in the dictionary

Kinda the same way Bomp added Rizz and such despite being a really silly word

**Comment by JaxenXK** [comment]
theyre widely used terms tho, so i dont rly see the point of having them removed

**Comment by JaxenXK** [comment]
..unless they're trend words

**Comment by KisaragiDayo** [comment]
> ..unless they're trend words

Does that mean I can suggest those silly "trend" words when they finally get exhausted?

**Comment by iminlikewithyou** [comment]
you're too focused within the lens of an internet user.

they're specialized words outside of general usage.  for an internet term like “pogchamp” to be included, it has to not only be widely used outside of its specific area (twitch), but also increasingly in general discourse.

general discourse would typically mean people in real life. you can use it in typical conversations.

i'm not saying remove it from the dictionary entirely. i'm saying that these words are far too specialized and should be in a separate section. i'm the one who added it and it was because i lacked the ability to see that these words are only known by a specific subculture of the internet.

it is the same way scientific words shouldn't be added to the main dictionary and should also be split.

**Comment by iminlikewithyou** [comment]
> Kinda the same way Bomp added Rizz and such despite being a really silly word

being a silly word doesn't constitute that it is now used in general discourse and not simply an internet word - this is a far different kind of word imo

**Comment by Shrenne** [comment]
> you're too focused within the lens of an internet user.
> 
> they're specialized words outside of general usage. for an internet term like “pogchamp” to be included, it has to not only be widely used outside of its specific area (twitch), but also increasingly in general discourse.
> 
> general discourse would typically mean people in real life. you can use it in typical conversations.
> 
> i'm not saying remove it from the dictionary entirely. i'm saying that these words are far too specialized and should be in a separate section. i'm the one who added it and it was because i lacked the ability to see that these words are only known by a specific subculture of the internet.
> 
> it is the same way scientific words shouldn't be added to the main dictionary and should also be split.

I disagree. As long as words are widely circulated enough, even in a specific field, it should be considered as words in the main dictionary. Being in a specialized field does not mean they should only be included in dictionaries specific to the field, as demonstrated by most dictionaries having highly scientific words.

For this instance, I am certain most people have seen the words POG and POGCHAMP being used outside of Twitch on other social media, or even in real life speech when said by streamers. For example, our OMG Community server alone has 1,440 results of the word Pog, and I would like to bring up that over half the world's population uses the internet, making the specialized field extremely vast and inclusive.

**Comment by iminlikewithyou** [comment]
> As long as words are widely circulated enough, even in a specific field, it should be considered as words in the main dictionary. Being in a specialized field does not mean they should only be included in dictionaries specific to the field, as demonstrated by most dictionaries having highly scientific words.

widely circulated enough = used in general discourse

dictionaries do add scientific words that have extended into general discourse. that is why they do have scientific words in the dictionary, but not all words are like this.
